WIP Google Consent V2 (#14)
run-converge/docs-mintlify: guides/google-consent-v2.mdx

---
title: "Transitioning to Google Consent V2"
description: "How to navigate the transition to the new Google consent mode"
---

<Accordion title="Key takeaways" icon="lightbulb" iconType="duotone">

>   - With the DMA going into effect, Google introduced two changes impacting advertisers on Google Ads.

Reviewer: ```suggestion
  - With the Digital Markets Act (DMA) going into effect, Google introduced two changes impacting advertisers on Google Ads.
```

  
    1. Publishers are required to collect the user's consent and pass it on to Google.
    2. The user's consent is broken down into two subcategories (Consent Mode V2): advertising purposes and personlized advertising purposes.

  - The Converge connections are compatible with any consent implementation, either through a CMP platform, a custom implementation, or default setup.

>   - It is still unclear how and whether data without consent will be processed.

Reviewer: ```suggestion
  - As of early March, it is still unclear how and whether data without consent will be processed.
```

  - Consent limits the data collection to GA4 and Google Ads. Google will try to fill the gap by modelling missing data. This will lead to less accurate reporting. 
</Accordion>

## Introduction
Over the past months, Google has updated their policies to ensure compliance with the European Digital Markets Act (DMA). Two major changes impact advertisers on Google Ads. -- Publishers on Google Ads are required to collect user consent on their website

> Google now requires web owners advertising on Google Ads to collect explicit visitor consent for sharing data with Google. User consent should be passed to the tags installed on the website.

Reviewer: ```suggestion
Google now requires businesses advertising on Google Ads to collect explicit visitor consent for sharing data with Google. User consent should be passed to the tags installed on the website.
```
